src/site/_includes/macros/tagschip.njk

```diff
@@ -0,0 +1,14 @@
+{#
+  @typedef {string[]} tags - An array of tag keys to display.
+  @property {string} [className] - A space separated list of classes to apply.
+#}
+{% macro tagschip(tags, className = '') %}
```

Comment:
Would you mind camel casing this and all other macros. I think we do that for other macros on the site like audioPlayer() and audioFab().

Also, could you rename this to tagChips instead of tagsChip. I think the UI element that we're rendering is called a chip, and we're rendering a set of them so—"tag chips".